docs/development.md

```diff
@@ -87,6 +86,10 @@ Yay, the plugin was loaded!
 
 The `dev` script will start your app using [nodemon](https://github.com/remy/nodemon#nodemon), which will watch for any files changes in your local development environment and automatically restart the server.
 
+**Other available scripts**
+* `$ npm start` to start your app without watching files.
+* `$ npm run lint` to lint your code using [standard](https://www.npmjs.com/package/standard).
```

Comment:
We could also add "Use `npm run` to see available scripts"

Reply:
In that case we won't be able to mention what exactly the command do. Like will lint make changes in the code base or just prompt about the issues.

Reply:
I think it would be more thorough to go with @zeke's suggestion, so that in case the scripts change in the future, we don't have to remember to edit this list every time.

Reply:
:+1: